src/elisp/treemacs-extensions.el

```diff
@@ -193,7 +192,8 @@ MORE-PROPERTIES is a plist of text properties that can arbitrarily added to the
 node for quick retrieval later."
   (treemacs-static-assert (and icon label-form state key-form)
     "All values except :more-properties and :face are mandatory")
-  `(list prefix ,icon
```

Comment:
If root-level nodes are printed, don't print prefix but print the project separator as a suffix instead.